mavros_msgs/CMakeLists.txt

```diff
@@ -1,14 +1,15 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(mavros_msgs)
 
-find_package(catkin REQUIRED COMPONENTS geometry_msgs message_generation std_msgs)
+find_package(catkin REQUIRED COMPONENTS geometry_msgs sensor_msgs message_generation std_msgs)
 
 include_directories(include)
 
 add_message_files(
   DIRECTORY msg
   FILES
   ActuatorControl.msg
+  ActuatorControlTarget.msg
```

Comment:
will replace for `ActuatorControlTarget` to keep the logic from the MAVLINK common message set.